release_notes/368-release-notes.adoc

```diff
@@ -59,18 +59,19 @@ In previous versions of {product-title}, the `write` permission for the `APIToke
-- Default behavior of `roxctl helm output` results now in using container images from `registry.redhat.io` instead of `stackrox.io`.
 - By default, notifications will be sent for every runtime policy violation instead of only the first encountered violation. If this is undesired, setting an environment variable `NOTIFY_EVERY_RUNTIME_EVENT` to `false` will restore the previous behavior. Please note that the environment variable will be removed in a future release, so please notify the ACS team if you have a valid use case.
 - Certain ACS images were moved to new repositories:
-  - main: from `registry.redhat.io/rh-acs/main` to `registry.redhat.io/advanced-cluster-security/rhacs-main-rhel8`
-  - collector: from `registry.redhat.io/rh-acs/collector` (with `-latest` tag) to `registry.redhat.io/advanced-cluster-security/rhacs-collector-rhel8`
-  - collector (slim): from `registry.redhat.io/rh-acs/collector` (with `-slim` tag) to `registry.redhat.io/advanced-cluster-security/rhacs-collector-slim-rhel8`
-  - scanner: from `registry.redhat.io/rh-acs/scanner` to `registry.redhat.io/advanced-cluster-security/rhacs-scanner-rhel8`
-  - scanner-db: from `registry.redhat.io/rh-acs/scanner-db` to `registry.redhat.io/advanced-cluster-security/rhacs-scanner-db-rhel8`
+** main: from `registry.redhat.io/rh-acs/main` to `registry.redhat.io/advanced-cluster-security/rhacs-main-rhel8`
+** collector: from `registry.redhat.io/rh-acs/collector` (with `-latest` tag) to `registry.redhat.io/advanced-cluster-security/rhacs-collector-rhel8`
+** collector (slim): from `registry.redhat.io/rh-acs/collector` (with `-slim` tag) to `registry.redhat.io/advanced-cluster-security/rhacs-collector-slim-rhel8`
+** scanner: from `registry.redhat.io/rh-acs/scanner` to `registry.redhat.io/advanced-cluster-security/rhacs-scanner-rhel8`
+** scanner-db: from `registry.redhat.io/rh-acs/scanner-db` to `registry.redhat.io/advanced-cluster-security/rhacs-scanner-db-rhel8`
 - Tags of `scanner`, `scanner-db`, and `collector` (including slim variant) images are now identical to the tag of `main` image (same as product version) for the released images. For example, a scanner image for ACS 3.68.0 is now identified as following `registry.redhat.io/advanced-cluster-security/rhacs-scanner-rhel8:3.68.0` and `stackrox.io/scanner:3.68.0`. Please make sure you follow this versioning scheme when upgrading manually. This scheme will be used for all future releases.
 - Collector Slim image name and tag have changed. Now the `-slim` is not part of the image tag but part of the image name. This means that Collector Slim image for the release 3.68.0 is identified as `registry.redhat.io/advanced-cluster-security/rhacs-collector-slim-rhel8:3.68.0` and `collector.stackrox.io/collector-slim:3.68.0`.
+- Scanner DB image at `registry.redhat.io/advanced-cluster-security/rhacs-scanner-db-rhel8` is now based on `rhel8/postgresql-12`.
```

Comment:
@misberner This was not in stackrox CHANGELOG because this is coming from downstream images. I still thought it may be useful to point out in case someone would run into entitlement issues of rhel8.
What do you think?

Reply:
This isn't really relevant to a customer. Its more important from our perspective. They aren't running their own postgres deployment or having their DBA manage it AFAIK.

Reply:
Hmm - do you think the entitlement issue is likely? We were able to run downstream images with no issues, in spite of no entitlements. I don't think it hurts to call this out, but I also don't see why a customer would care

Reply:
Well, it may be a concern for non-OpenShift clusters because from my understanding RHEL containers will not like to run on non-RHEL hosts because of entitlements. In this case this is for customers who deploy Central on non-OpenShift which should be only exceptional cases but I've heard such cases still exist.

Reply:
If this is true, we cannot roll out the new downstream images to everyone - we certainly don't want people to go back to `stackrox.io`. We need to test downstream images for at least scanner DB on a non-OpenShift cluster (GKE or so) ASAP to verify if that matters.

Reply:
For context, pretty much all of our pre-acquisition customers are running Central on non-OpenShift under grandfathered support terms.

FWIW, I can run the `postgresql-12` container with no issue on my Mac with a linuxkit-based VM:
```
$ docker run -e POSTGRESQL_USER=malte -e POSTGRESQL_PASSWORD=secret -e POSTGRESQL_DATABASE=default registry.redhat.io/rhel8/postgresql-12
The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.
[...]
Success. You can now start the database server using:

    pg_ctl -D /var/lib/pgsql/data/userdata -l logfile start

waiting for server to start....2022-01-28 11:23:56.607 UTC [34] LOG:  starting PostgreSQL 12.9 on x86_64-redhat-linux-gnu, compiled by gcc (GCC) 8.5.0 20210514 (Red Hat 8.5.0-4), 64-bit
[...]
 done
server started
/var/run/postgresql:5432 - accepting connections
[...]
```

Reply:
Well, after trying to comprehend these docs
* https://developers.redhat.com/articles/ubi-faq
* https://access.redhat.com/solutions/5433891

I realize I may be wrong in my terminology and not entirely correct in my understanding but what I mean is this
![image](https://user-images.githubusercontent.com/537715/151539073-0aa9bd80-72ac-488e-a4b4-c7f2a2e119d5.png)
That's our case and there might be implications for customers that I don't know. Therefore wanted to call that out explicitly.

Reply:
Test results show that scanner-db starts up and functions fine on k8s. See https://srox.slack.com/archives/C02HX541YQN/p1643369429309329
Kudos to Fred